README.md

```diff
@@ -27,16 +31,16 @@ What is the purpose of the `alt` attribute? e.g:
 <img src='https://jesse.sh/img/me.jpg' alt='Jesse Shawl'/>
 ```
 
-```text
-Your answer here.
+```
+If an image can’t be displayed, the alt attribute text describes what the image would have been.
```

Comment:
:+1:
Most technical interviewers would be looking for you to discuss accessibility here.